styles/desktop.css

```diff
@@ -0,0 +1,345 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+body {
+  padding: 0;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+  background-color: #f4f5f7;
+}
+
+.header {
+  display: grid;
+  grid-template-columns: 1fr;
+  background-color: #fff;
+  border-radius: 0 0 0 100px;
+  -webkit-border-radius: 0 0 0 100px;
+  -moz-border-radius: 0 0 0 100px;
+  -ms-border-radius: 0 0 0 100px;
+  -o-border-radius: 0 0 0 100px;
+}
+
+.social-icons {
+  display: flex;
+  gap: 20px;
+}
+
+.header .social-icons {
+  padding-bottom: 225px;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 25px 155px;
+}
+
+.intro {
+  background-image: url("../assets/desktop_images/Header\ bg.png");
+  background-repeat: no-repeat;
+  object-fit: cover;
+  background-position-x: center;
+}
+
+.introc-content {
+  padding: 359px 260px 0 260px;
+}
+
+.introc-content h1 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 60px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.37px;
+  color: #091e42;
+}
+
+.introc-content p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #344563;
+}
+
+.introc-content h3 {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+
+  /* identical to box height, or 150% */
+
+  /* Primary/B300 */
+
+  color: #7f8cff;
+}
+
+/* work */
+.works-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.card-wrapper {
+  display: flex;
+  margin: 142px;
+  padding: 24px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 24px;
+}
+
+.card-wrapper:nth-child(even) {
+  flex-direction: row-reverse;
+}
+
+.card-content-wrapper {
+  margin: 46px;
+}
+
+.card-wrapper img {
+  width: 544px;
+}
+
+.card-wrapper h3 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+}
+
+.see-projects-btn {
+  margin-top: 40px;
+  padding: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: rgba(64, 83, 252, 1);
+  background: rgba(57, 109, 242, 0);
+  border: 2px solid rgba(64, 83, 252, 1);
+  border-radius: 10px;
+  -webkit-border-radius: 10px;
+  -moz-border-radius: 10px;
+  -ms-border-radius: 10px;
+  -o-border-radius: 10px;
+  transition: background-color 1s, color 0.5s, rotate 0.8s ease-in-out;
+  -webkit-transition: background-color 1s, color 0.5s, rotate 0.8s ease-in-out;
+  -moz-transition: background-color 1s, color 0.5s, rotate 0.8s ease-in-out;
+  -ms-transition: background-color 1s, color 0.5s, rotate 0.8s ease-in-out;
+  -o-transition: background-color 1s, color 0.5s, rotate 0.8s ease-in-out;
+}
+
+.see-projects-btn:hover {
+  background-color: teal;
+  color: white;
+  rotate: 10deg;
+  border: none;
+}
```

Comment:
- I suggest using the right hover effect as provided by Microverse on the Figma design, while also adding an active effect:
Your hover effect styles:
![Screenshot from 2023-04-07 11-20-14](https://user-images.githubusercontent.com/97834160/230583097-270df217-50fb-44b8-a5d2-1aff383f3160.png)

Figma:

![Screenshot from 2023-04-07 11-22-30](https://user-images.githubusercontent.com/97834160/230583176-99658296-a9e5-4d50-84b3-6d3f4882de8d.png)

Reply:
Thank you.
Will fix this